github.go

```diff
@@ -5,21 +5,21 @@ import (
 	"crypto/tls"
 	"errors"
 	"fmt"
+	"github.com/google/go-github/v32/github"
+	"github.com/shurcooL/githubv4"
+	"golang.org/x/oauth2"
 	"io"
 	"net/http"
 	"net/url"
 	"os"
-
-	"golang.org/x/oauth2"
-
-	"github.com/google/go-github/github"
 )
 
 // Last run with counterfeiter v6
 //go:generate counterfeiter -o fakes/fake_git_hub.go . GitHub
 
 type GitHub interface {
 	ListReleases() ([]*github.RepositoryRelease, error)
+	ListReleasesV4() ([]*github.RepositoryRelease, error)
```

Comment:
This doesn't need to be on the interface, as it's only ever called internally

Reply:
ok let me remove this, I also was thinking if it's necessary or not, but decided to declare in the interface to have more consistency.